packages/front-end/components/Dimensions/DimensionChooser.tsx

```diff
@@ -78,9 +78,17 @@ export default function DimensionChooser({
   // Date is always available
   const builtInDimensions = [
     {
-      label: "Date",
+      label: "Time Series (Date Exposed Cohort)",
```

Comment:
Is this option still useful?  Should we hide it from the list?

Reply:
I'm not sure we should hide it. I think there is still some use to it, but maybe I would place it below the other time series. I think we can finalize this once we merge with Bryce's FE design, and I'd prefer to make Cumulative and Daily below just be in one view, but that might have to wait for the snapshot refactor. Maybe we should just launch Cumulative until then.

Reply:
We decided to keep that for now, and we will add a separate way in to the cumulative (and from there a toggle to daily) TS. This will happen in a follow up PR as part of the work on improving the results UI.